[14] Log opened at 2024-07-10 18:56:46.189540
[14] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.14'
[14] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[14] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[14] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[14] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[32] Log opened at 2024-07-10 18:56:49.603676
[32] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.32'
[32] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[32] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[32] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[32] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[32] Log closed at 2024-07-10 18:56:49.828428

[14] Log closed at 2024-07-10 18:56:49.845931

[14] Log opened at 2024-07-10 18:59:11.205525
[14] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.14'
[14] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[14] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[14] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[14] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[32] Log opened at 2024-07-10 18:59:14.138730
[32] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.32'
[32] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[32] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[32] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[32] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[32] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[32] Log closed at 2024-07-10 18:59:14.348684

[14] Log closed at 2024-07-10 18:59:14.363590

[15] Log opened at 2024-07-10 19:14:36.053698
[15] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.15'
[15] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[15] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[15] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[15] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[33] Log opened at 2024-07-10 19:14:39.535932
[33] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.33'
[33] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[33] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[33] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[33] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[33] Log closed at 2024-07-10 19:14:39.755162

[15] Log closed at 2024-07-10 19:14:39.774624

[7] Log opened at 2024-07-10 19:19:26.671233
[7] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.7'
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] Log closed at 2024-07-10 19:19:27.647651

[8] Log opened at 2024-07-10 19:19:52.143224
[8] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.8'
[8] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[8] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[8] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[8] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[8] Log closed at 2024-07-10 19:19:52.524583

[7] Log opened at 2024-07-10 19:31:42.409661
[7] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.7'
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] Log closed at 2024-07-10 19:31:42.660124

[8] Log opened at 2024-07-10 19:32:40.839430
[8] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.8'
[8] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[8] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[8] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[8] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[8] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[8] Log closed at 2024-07-10 19:32:41.121979

[7] Log opened at 2024-07-10 19:33:38.039502
[7] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.7'
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[7] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[7] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[7] Log closed at 2024-07-10 19:33:38.308558

[59] Log opened at 2024-07-10 19:35:48.952353
[59] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.59'
[59] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[59] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[59] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[59] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[59] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[59] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[59] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[59] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[59] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[59] Log closed at 2024-07-10 19:35:49.338997

[15] Log opened at 2024-07-10 19:37:26.923598
[15] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.15'
[15] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[15] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[15] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[15] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[33] Log opened at 2024-07-10 19:37:30.362269
[33] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.33'
[33] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[33] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[33] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[33] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[33] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[33] Log closed at 2024-07-10 19:37:30.592585

[15] Log closed at 2024-07-10 19:37:30.614097

[38] Log opened at 2024-07-10 19:37:36.758873
[38] [Config] INFO: Control socket set up succesfully: '@xdebug-ctrl.38'
[38] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[38] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[38] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[38] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[38] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[38] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[38] [Step Debug] INFO: Connecting to configured address/port: host.docker.internal:9003.
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', poll success, but error: Operation now in progress (29).
[38] [Step Debug] WARN: Creating socket for 'host.docker.internal:9003', connect: Network is unreachable.
[38] [Step Debug] ERR: Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
[38] Log closed at 2024-07-10 19:37:37.085613

